### PR TITLE
refactor: form entities from submission

### PR DESCRIPTION
**Pull request by theus77**
|Q              |A  |
|---------------|---|
|Bug fix?       |y|
|New feature?   |n|	
|BC breaks?     |n|	
|Deprecations?  |n|	
|Fixed tickets  |n|	

With the integration of the submission bundle in elasticms we are facing conflict between the Core's FormSubmission entities and the Submission's one. This PR is about adding a dependency to SubmissionBundle and use Submission's entities. 
Fix the doctrine:migrate:diff command

### CODE REVIEW

**Comment by theus77** [comment]
I have tried to have inheritance from Submission entities (but it's really complex). So the less code and pragmatic approach wins

File: src/Form/Data/EntityRow.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 namespace EMS\CoreBundle\Form\Data;
 
-use EMS\CoreBundle\Entity\EntityInterface;
+use EMS\SubmissionBundle\Entity\EntityInterface;
```
Comment (Davidmattei) [inline src/Form/Data/EntityRow.php:7]: I do not like this, this is the EntityRow for all entity datatables. And It now uses the SubmissionBundle entityInterface allways?

**Review by Davidmattei — CHANGES_REQUESTED** [review]

File: src/Form/Data/EntityRow.php (hunk 1 above)
Comment (Davidmattei) [reply src/Form/Data/EntityRow.php:7]: And the src/Entity/EntityInterface.php  is extension of EMS\SubmissionBundle\Entity\EntityInterface

Comment (theus77) [reply src/Form/Data/EntityRow.php]: It extends a common interface now

**Review by Davidmattei — APPROVED** [review]

**Pull request merged**